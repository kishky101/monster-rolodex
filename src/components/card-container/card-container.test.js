import React from 'react';
//import {render, screen} from '@testing-library/react';
import renderer from 'react-test-renderer';
import CardContainer from './card-container.component';

test('testing snapshot', () => {
    const cardcontSnap = renderer.create(<CardContainer monster={{id: 2, name: 'Ervin Howell', username: 'Bretin', email: 'Shanna@melissa.tv'}} />).toJSON;
    expect(cardcontSnap).toMatchSnapshot();
})