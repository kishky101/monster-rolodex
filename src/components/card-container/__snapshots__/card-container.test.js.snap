// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing snapshot 1`] = `[Function]`;
